Makefile

```diff
@@ -89,27 +89,14 @@ welcome:
 	@printf "\033[36m               |___/|_|                  \n"
 	@printf "\033[m\n"
 
-install: node_modules
+install:
+	@$(NPM) run bootstrap
+	@$(NPM) install
 
 # Simply running `make run` will spawn the Node.js server instance.
 run: welcome githooks install build
 	@$(NODE) build/bundle-$(CALYPSO_ENV).js
 
-node-version:
-	@$(BIN)/check-node-version
-
-# a helper rule to ensure that a specific module is installed,
-# without relying on a generic `npm install` command
-node_modules/%: | node-version
-	@$(NPM) install $(notdir $@)
-
-# ensures that the `node_modules` directory is installed and up-to-date with
-# the dependencies listed in the "package.json" file.
-node_modules: package.json | node-version
```

Comment:
Killing this forces us to re-run npm install every time we run. npm install is really slow and we'd like to avoid it when we don't have to run it.